gui/src/exception/ArgsException.cpp

```diff
@@ -10,6 +8,6 @@
 using namespace GUI;
 
 ArgsException::ArgsException(const std::string &message)
```

Comment:
Just fyi, what I meant was actually this:
```suggestion
ArgsException::ArgsException(std::string message): AException(std::move(message), "ArgsException")
```

But it's ok, just you have time fix this ^^